charts/spiderpool/values.yaml

```diff
@@ -67,12 +67,14 @@ ipam:
       ## @param ipam.gc.gcAll.intervalInSecond the gc all interval duration
       intervalInSecond: 600
 
-    GcDeletingTimeOutPod:
-      ## @param ipam.gc.GcDeletingTimeOutPod.enabled enable retrieve IP for the pod who times out of deleting graceful period
-      enabled: true
+    ## @param ipam.gc.enableGcStatelesTerminatingPodOnReadyNode enable reclaim IP for the stateless pod that times out of deleting graceful period with its node ready
+    enableGcStatelesTerminatingPodOnReadyNode: true
 
-      ## @param ipam.gc.GcDeletingTimeOutPod.delay the gc delay seconds after the pod times out of deleting graceful period
-      delay: 0
+    ## @param ipam.gc.enableGcStatelesTerminatingPodOnNotReadyNode enable reclaim IP for the stateless pod who that out of deleting graceful period with its node not ready
```

Comment:
who that out of deleting graceful period with its node not ready
--------->
who is over deleting graceful period on a not-ready node